cost/schedule_instances/schedule_instances.pt

```diff
@@ -0,0 +1,307 @@
+name "Schedule instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that start and stops instances based on a schedule.
+See [README](https://github.com/rightscale/policy_templates/cost/schedule_instances) for more details"
+long_description "Version 1.0"
+severity "medium"
+category "Cost"
+
+
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses"
```

Comment:
I think we can use the same label name as other policies. I went ahead and made them consistent a few weeks ago if it helps.

Reply:
good catch.  I updated this.